src/style.css

```diff
@@ -1 +1,50 @@
-/* Styles go here */
+.header {
+  margin: 0;
+  display: flex;
+  width: 100%;
+  flex-direction: row;
+  justify-content: space-around;
```

Comment:
```suggestion
  justify-content: space-between;
```
лого і навігація повинні бути рознесені по краям

Reply:
так ніби ж в референсах вони не по краям?

Reply:
бокові падінги це було перше, що я спробувала, але тоді в тесті пропорції чомусь дуже міняються і лінк "відео" взагалі пропадає (не в браузері)